docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -2,6 +2,21 @@
 Pulp Puppet 2.8 Release Notes
 =============================
 
+Pulp Puppet 2.8.4
+=================
+
+New Behaviours
+--------------
+
+From now on name of the puppet module will correspond to the name specified in its metadata,
+i.e will be in the format of 'name/author' or 'name-author'.
```

Comment:
Puppet does not actually enforce a convention here so we should remove these examples since it could be anything really.